feed/utils.js

```diff
@@ -0,0 +1,50 @@
+async function getActivityFeedData(query = {}, nextLink) {
```

Comment:
can we validate the query object to ensure it contains expected parameters before constructing the URL.

Reply:
Noted.